docs/_includes/asciidoc-vs-markdown.adoc

```diff
@@ -0,0 +1,259 @@
+////
+A comparison between asciidoc and markdown.
+This file is included in the user-manual document
+////
+
+The most compelling reason in choosing a lightweight markup is minimizing the technical concepts that content authors must grasp in order for them to be immediately productive.
+
+A popular choice is Markdown. The main advantage of Markdown lies in its small syntax: its manual could be its own cheatsheet. But this same advantage is also its greatest fault: as soon as authors need something slightly more complex (a table, footnotes, including YouTube videos, ecc) they find themselves manually typing in raw html, or looking for extensions to the language (so that Markdown has now become a mess of different implementations, called "`flavours`", that prevent it to be truly universal).
+
+AsciiDoc is an excellent alternative in that it offers a syntax more concise than (or as concise as) Markdown, while being by far more flexible and more powerful without requiring the use of HTML or "`flavors`" to get essential syntax like tables, definition lists, admonitions (tips, notes, warnings, etc) and table of contents.
+
+==== A basic comparison by example
+
+[cols=3]
+|===
+| Formatting
+| Markdown
+| AsciiDoc
+
+| Bold (word)
+a|
+[source, markdown]
+----
+**bold**
+----
+a|
+[source, asciidoc]
+----
+*bold*
+----
+
+| Bold (first letter)
+a|
+[source, markdown]
+----
+**b**old
+----
+a|
+[source, asciidoc]
+----
+**b**old
+----
+
+| Italic (word)
+a|
+[source, markdown]
+----
+*italic*
+----
+a|
+[source, asciidoc]
+----
+_italic_
+----
+
+| Italic (first letter)
+|n/a
+a|
+[source, asciidoc]
+----
+__i__talic
+----
+
+| Literal (word)
+a|
+[source, markdown]
+----
+`literal`
+----
+a|
+[source, asciidoc]
+----
++literal+ or `literal`
+----
+
+| Literal (first letter)
+a|
+[source, markdown]
+----
+`l`iteral
+----
+a|
+[source, asciidoc]
+----
+++l++iteral
+----
+
+| Link with text
+a|
+[source, markdown]
+----
+[AsciiDoc](http://asciidoc.org)
+----
+a|
+[source, asciidoc]
+----
+http://asciidoc.org[AsciiDoc]
+----
+
+| Image w/ Alt Text
+a|
+[source, markdown]
+----
+![Logo](/images/logo.png)
+----
+a|
+[source, asciidoc]
+----
+image:logo.png[Logo]
```

Comment:
Make it clear that this is an inline image. We may want another entry for "Block Image w/ Alt Text" and say that there is no equivalent in Markdown.